### PR TITLE
noto-fonts-emoji: 2.034 -> 2.038  [Unicode 15.0]

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
@ofborg build noto-fonts-emoji

**Comment by sternenseemann** [comment]
Tested all reverse deps locally on x86_64-linux:

```
5 packages built:
deltachat-desktop noto-fonts-emoji python310Packages.nototools python39Packages.nototools twitter-color-emoji
```

File: pkgs/data/fonts/noto-fonts/default.nix [hunk 1]
```diff
@@ -171,8 +171,10 @@ in
     enableParallelBuilding = true;
 
     installPhase = ''
+      runHook preInstall
       mkdir -p $out/share/fonts/noto
-      cp NotoColorEmoji.ttf fonts/NotoEmoji-Regular.ttf $out/share/fonts/noto
```
Comment (Mathnerd314) [inline pkgs/data/fonts/noto-fonts/default.nix:175]: This seems right. There is a new black-and-white emoji font, discussed in https://github.com/googlefonts/noto-emoji/issues/390, but if that gets packaged it should probably be a separate package.

**Review by Mathnerd314 — APPROVED** [review]

File: pkgs/data/fonts/noto-fonts/default.nix (hunk 1 above)
Comment (sternenseemann) [reply pkgs/data/fonts/noto-fonts/default.nix:175]: Good to know! Would be nice to be able to build that from source then, as well.

**Comment by sternenseemann** [comment]
@Mathnerd314 Is it just me or are the new Unicode 15 emojis like shaking face, pink heart etc. missing despite being present in the source? Wonder what's going wrong there (for me).

**Comment by Mathnerd314** [comment]
No idea, could be harfbuzz or freetype or something else in the font rendering stack that hasn't updated the database. I update my system like every 6 months so I wouldn't know.

**Pull request merged**